Banking/src/main/java/com/example/Banking/exception/IdNotFoundException.java

```diff
@@ -0,0 +1,21 @@
+package com.example.Banking.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+@Component
```

Comment:
We don't need @Component here as an exception is something we create new object every time.